src/Commands/LivewireDestroyCommand.php

```diff
@@ -27,7 +27,7 @@ public function handle()
 
         if (! $force = $this->option('force')) {
             $shouldContinue = $this->confirm(
-                "Are you sure you want to delete the following files?\n\n{$this->parser->classPath()}\n{$this->parser->viewPath()}\n"
+                "Are you sure you want to delete the following files?"
```

Comment:
Why remove this from the output? I want to show the user what files they are about to delete

Reply:
They were just trying to get the tests passing for CI. A PR was already merged that maintains the file output, and this PR should be caught up with that (https://github.com/calebporzio/livewire/pull/73/files/d057420a199eab70f524ef6a1c19d5b1525c3880..043abf410befb6f6babd0881119b781f38e35c54#diff-244259b80d8e7e333e1448a92aae04f0R22).

Reply:
Hey @Caleb,

this was done just to keep the tests green for initial CI setup, but as @loganhenson has fixed the DestroyCommandTest in [#88](), we should keep command as it were. Missed it when resolving stuff in a hurry this morning.

Thanks,
g